app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to controller: "users", action: "show", id: @user.id
```

Comment:
em có thể viết ngắn gọn lại như này nhé:
```suggestion
      redirect_to @user
```

Reply:
em thay bằng @user thì lúc tạo tài khoản xong nó vẫn lưu vào database nhưng không chuyển đến được trang hiển thị người dùng ạ nên phải thay như trên ạ
![image](https://user-images.githubusercontent.com/43948239/55871410-bfb40e00-5bb4-11e9-9229-cd915d34469c.png)